server/service/src/purchase_order_line/update/mod.rs

```diff
@@ -91,7 +91,7 @@ pub fn update_purchase_order_line(
                         status: Some(PurchaseOrderStatus::Confirmed),
                         ..Default::default()
                     };
-                    let _ = update_purchase_order(ctx, store_id, input, user_has_permission);
+                    let _ = update_purchase_order(ctx, store_id, input, user_has_auth_permission);
```

Comment:
trying to remember - thinking that this should just work without the let definition?